app/views/search.nunjucks

```diff
@@ -15,6 +15,9 @@
         <!-- label -->
       </label>
 
+      {% if errorMessage %}
```

Comment:
Dreaded back button behaviour:

Enter invalid/missing postcode -> error message displayed -> enter valid postcode -> results listed -> hit browser back button -> valid postcode displayed in textbox + error message displayed.